queries.sql

```diff
@@ -35,4 +35,21 @@ SELECT species.name, COUNT(*) FROM animals JOIN species ON animals.species_id =
 SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name LIKE '%Jennifer Orwell%' AND animals.species_id = 2;
 SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.fu
 ll_name LIKE '%Dean Winchester%' AND animals.escape_attempts = 0;
-SELECT owners.full_name AS "Owner Name", COUNT(animals.name) AS "Num of Animals" FROM owners JOIN animals ON owners.id = animals.owner_id GROUP BY owners.full_name ORDER BY COUNT(animals.name) DESC LIMIT 1;
\ No newline at end of file
+SELECT owners.full_name AS "Owner Name", COUNT(animals.name) AS "Num of Animals" FROM owners JOIN animals ON owners.id = animals.owner_id GROUP BY owners.full_name ORDER BY COUNT(animals.name) DESC LIMIT 1;
+
+SELECT vets.name AS "Vet Name", animals.name AS "Animal Name", visits.date FROM vets JOIN visits ON vets.id = visits.vet_id JOIN animals ON animals.id = visits.animal_id WHERE vets.name LIKE '%William Tatcher%' GROUP BY vets.name, animals.name, visits.date ORDER BY visits.date DESC LIMIT 1;
+SELECT COUNT(animals.name) AS "Stephanie Mendez's Animals" FROM animals JOIN visits on animals.id = visits.animal_id JOIN vets ON vets.id = visits.vet_id WHERE vets.name LIKE '%Stephanie Mendez%';
+
+SELECT vets.name AS "Vet Name", species.name AS "specialization" FROM vets FULL OUTER JOIN specializations ON vets.id = specializations.vets_id FULL OUTER JOIN species ON species.id = specializations.species_id GROUP BY vets.name, species.name;
+
+SELECT animals.name AS "Animal Name", visits.date AS "Date" FROM animals JOIN visits ON animals.id = visits.animal_id JOIN vets ON vets.id = visits.vet_id WHERE vets.name LIKE '%Stephanie Mendez%' AND visits.date BETWEEN '2020-04-01' AND '2020-08-30' GROUP BY animals.name, visits.date;
+
```

Comment:
- [ ] Great job so far. The queries work as expected, but they give difficulties to read. Kindly add comments to the queries, identifying the query questions you are answering. Great work 👍🏾